database/src/main/scala/cromwell/database/sql/CallCachingStore.scala

```diff
@@ -7,7 +7,7 @@ import scala.concurrent.{ExecutionContext, Future}
 trait CallCachingStore {
   def addToCache(metaInfo: CallCachingResultMetaInfoEntry, hashes: Iterable[HashKeyAndValue], result: Iterable[ResultSimpleton])(implicit ec: ExecutionContext): Future[Unit]
   def metaInfoIdsMatchingHashes(hashKeyValuePairs: Set[HashKeyAndValue])(implicit ec: ExecutionContext): Future[Set[MetaInfoId]]
-  def fetchCachedResult(metaInfoId: MetaInfoId)(implicit ec: ExecutionContext): Future[Option[CachedResult]]
+  def fetchCachedResult(metaInfoId: MetaInfoId)(implicit ec: ExecutionContext): Future[CachedResult]
```

Comment:
ToL: Hmmm... although I agree in the one place that you're using it that you'll never get a miss, do we really want "otherwise it throws an exception" as part of the "generic Call Caching store" signature?

(Sorry to keep returning to this. I guess you can ignore for expediency)